Corim cleanup (#392)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

# Example Verifier Algorithm {#sec-verifier-abstraction}

This document assumes that Verifier implementations may differ.

> To facilitate the description of normative Verifier behavior, this document describes the internal representation for an example Verifier and demonstrates how the data is used in the verification phases outlined in ({{sec-appraisal-procedure}}).

Reviewer: No need for parentheses in this case.

```suggestion
To facilitate the description of normative Verifier behavior, this document describes the internal representation for an example Verifier and demonstrates how the data is used in the verification phases outlined in {{sec-appraisal-procedure}}.
```

Author: ok, i will change

The Conceptual Message type determines which attributes are mandatory.

> ### Internal Representation of keys {#sec-ir-ext}

Reviewer 2: ```suggestion
### Internal Representation of Cryptographic Keys {#sec-ir-ext}
```

Verifiers are not required to use this as their internal representation.
For the purposes of this document, appraisal is described in terms of the above cited internal representation.

#### ACS Processing Requirements

The ACS contains the actual state of Attester's Target Environments (TEs).

> The ACS contains Evidence ECTs (from Attesters) and Endorsements ECTs

Reviewer 3: ```suggestion
The ACS contains Evidence ECTs (from Attesters) and Endorsement ECTs
```

(e.g. from `endorsed-triple-record`).

CoMID Reference Values will be matched against the ACS following the comparison rules in {{sec-match-condition-ect}}.
This document describes an example evidence structure which can be
matched against these Reference Values.

> Each entry within Endorsements ECTs uses the syntax of `endorsed-triple-record`.

Reviewer 3: ```suggestion
Each Endorsement ECT contains the environment and internal representation of `measurement-map`s as extracted from an `endorsed-triple-record`.
```
What is an entry in an ECT? I understand an entry as something in the attestation context, like ACS, ae, rv, etc.. This also isn't exactly using the same syntax for environment and element-list as an `endorsed-triple-record`, since we use internal representations.

Author: i am ok with this clarity, works for me!

> When an `endorsed-triple-record` is translated to Endorsements ECTs it

Reviewer 2: ```suggestion
When an `endorsed-triple-record` is transformed to Endorsements ECTs it
```

Reference Values are matched with ACS entries by iterating through the `rv` list.
For each `rv` entry, the `condition` ECT is compared with an ACS ECT, where the ACS ECT `cmtype` contains `evidence`.

> [^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/302

Reviewer 3: This issue is not yet addressed.
